### DISCUSSION TITLE
计算器操作数合法性

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#55] [Q&A] [answered]
ppt说支持的数据类型是浮点数，且精确到一位小数。那么(1.00+1)这样包含更多位小数的表达式是否合法？
1.，2.这样的表达方式是否合法？

**Comment by binbin00** [comment, marked as answer]
https://github.com/liusy58/NJU_AdvancedProgramming_SP22/discussions/4
